tests/std/tests/Dev09_126254_persistent_aux_allocators/test.cpp

```diff
@@ -98,6 +98,12 @@ void dump_map() {
     }
 }
 
+void test_gh_2362() {

```

Comment:
Could we have a one-line-ish description of what we're testing here so this function makes sense even if #2362 goes away?